src/cls/queue/cls_queue_src.cc

```diff
@@ -14,6 +14,9 @@ using ceph::bufferlist;
 using ceph::decode;
 using ceph::encode;
 
+const uint64_t page_size = 4096;
```

Comment:
in Stevens, 8192 usually wins out as a blocking factor over 4092, and that's in an era when 4K was a more common page size--but 4K is way better than 1K